### PR TITLE
Cleanup Facility Group Analytics and Graphics 

### PR DESCRIPTION
**Pull request by timmyjose**
For chore: https://www.pivotaltracker.com/story/show/166151057

### CODE REVIEW

File: app/controllers/analytics/districts_controller.rb [hunk 1]
```diff
@@ -32,6 +28,10 @@ def set_facilities
     @facilities = policy_scope(@organization_district.facilities).order(:name)
   end
 
+  def set_cache_key
+    @cache_key = ["view", @organization_district.analytics_cache_key(@from_time, @to_time)]
```
Comment (kitallis) [inline app/controllers/analytics/districts_controller.rb]: Don't need `view` here, the view `cache do` already adds this.

File: app/controllers/analytics/facilities_controller.rb [hunk 2]
```diff
@@ -34,6 +24,10 @@ def set_organization
     @organization = @facility.organization
   end
 
+  def set_cache_key
+    @cache_key = ["view", @facility.analytics_cache_key(@from_time, @to_time)]
```
Comment (kitallis) [inline app/controllers/analytics/facilities_controller.rb]: Don't need view here, the view cache do already adds this.

File: app/controllers/analytics/districts_controller.rb (hunk 1 above)
Comment (timmyjose) [reply app/controllers/analytics/districts_controller.rb]: Removed `set_cache_key`.

File: app/controllers/analytics/facilities_controller.rb (hunk 2 above)
Comment (timmyjose) [reply app/controllers/analytics/facilities_controller.rb]: Removed `set_cache_key`.

**Comment by timmyjose** [comment]
Removed the redundant `set_cache_key` methods. Ready for merge.

**Pull request merged**